snippets.json

```diff
@@ -16,7 +16,7 @@
   },
   "with": {
     "prefix": ["with"],
-    "body": ["with $0;"],
+    "body": ["with $1; $0"],
```

Comment:
please remove the space in `with $1;$0`. One can decide on having a newline or a space themselves.